website/client/components/tasks/user.vue

```diff
@@ -135,7 +135,7 @@
       }
   }
 
-  .dropdown-icon-item:hover .svg-icon, .dropdown-item.active .svg-icon {
```

Comment:
isn't this going to break the divs using the `dropdown-icon-item` class?

Reply:
It shouldn't in this component since their parent element has the `.dropdown-item` class, but it might break any other places this class was being used. I got confused and thought vue styles were namespaced to their components :grin:

I'll have a look at the other places these classes are used and see how they look.

Reply:
@mrbeardy they're scoped indeed (when defined as `<style scoped>` but in this case `.dropdown-item` comes from a child component and since the css here it's scoped it won't be applied. It's only applied to elements that appear in this component, not in children's html

Reply:
:thinking: strange, it seems to be allowing me to style the child element when using `.dropdown-item`.

I'll remove this change for now, it's not a huge change and I'd rather not have it accidentally break something somewhere since it's not a game-changing issue.